pyspark/bigdl/dataset/news20.py

```diff
@@ -19,17 +19,17 @@
 import os
 import sys
 
-NEWS20_URL = 'http://qwone.com/~jason/20Newsgroups/20news-19997.tar.gz'  # noqa
+NEWS20_URL = 'http://qwone.com/~jason/20Newsgroups/20news-18828.tar.gz'  # noqa
```

Comment:
so 18828 is the datasource without header and label?

Reply:
Yes